Automattic/jetpack
# Social Menu & Social Media Icons: Add support for SMS.

## Proposed changes:

This brings in the latest version of the Social Logos repo, including an SMS button.
It also updates our legacy features, Social Menu & Social Media Icons, to support that new button.

Related:

- https://github.com/Automattic/jetpack/pull/36037/
- https://github.com/Automattic/social-logos/pull/131/

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?
* No

## Testing instructions:

* Go to `yoursite.com/wp-content/plugins/jetpack/_inc/social-logos/social-logos.html`
* Check the new SMS logo
* Go to Jetpack > Settings and enable extra sidebar widgets.
* Switch to an old theme, like Twenty Ten.
* Go to Plugins > Add New and install the Classic Widgets plugin.
* In a functionality plugin, add the following:
```php
// Add social menu support
add_action( 'after_setup_theme', function () {
	add_theme_support( 'jetpack-social-menu', 'svg' );
} );
```
* Edit your theme and add the following, in `footer.php` for example
```php
<?php jetpack_social_menu(); ?>
```
* Go to Appearance > Customize > Menus
* Create a new menu with some links, e.g. `https://bsky.app/profile/wordpress.bsky.social` and `sms:0102030405`
* Assign that menu to the Social Menu location
* Go to Appearance > Widgets menu
* Drag the Social Icons widget to one of the active widget areas, and add some links again.
* Check how things look like on the frontend, for both your social menu and the Social Icons Widget you created.
* Check that the sharing buttons (added via Settings > Sharing) still work fine.
    * Make sure that those buttons also test well on WordPress.com Simple.
